orchagent/crmorch.cpp

```diff
@@ -457,10 +457,14 @@ void CrmOrch::getResAvailableCounters()
                 sai_status_t status = sai_switch_api->get_switch_attribute(gSwitchId, 1, &attr);
                 if (status != SAI_STATUS_SUCCESS)
                 {
-                    if(status == SAI_STATUS_NOT_SUPPORTED)
+                    if((status == SAI_STATUS_NOT_SUPPORTED) ||
+                       (status == SAI_STATUS_ATTR_NOT_SUPPORTED_0) ||
```

Comment:
can you use 

https://github.com/opencomputeproject/SAI/blob/master/inc/saistatus.h#L282

Reply:
and this one. https://github.com/opencomputeproject/SAI/blob/master/inc/saistatus.h#L272